pom.xml

```diff
@@ -267,6 +272,44 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack-schemas</id>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <phase>generate-sources</phase>
+                        <configuration>
+                            <artifact>org.apache.kafka:kafka-clients:3.1.0</artifact>
+                            <includes>common/message/*.json</includes>
+                            <outputDirectory>${project.build.directory}/message-schemas</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.kroxylicious</groupId>
+                <artifactId>kroxylicious-krpc-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-messages</id>
+                        <goals>
+                            <goal>generate-messages</goal>
+                        </goals>
+                        <configuration>
+                            <schemaDirectory>${project.build.directory}/message-schemas/common/message</schemaDirectory>
+                            <templateNames>Kproxy/Filter.ftl</templateNames>
+                            <outputFilePattern>${schemaName}Filter.java</outputFilePattern>
+                            <outputDirectory>${project.build.directory}/generated-sources/krpc/io/strimzi/kproxy/api/filter</outputDirectory>
```

Comment:
For now I'm specifying the full package name here. Eventually, only the top dir (".../krpc") should be needed, and the package name derived automatically by the generator.